content/guides/user-guides/delfin/api-clients/_index.md

```diff
@@ -0,0 +1,313 @@
+---
+title: Delfin Thirdparty Integration Guide
+description: "User guide for Delfin API Clients"
+weight: 30
+disableToc: false
+tags: ["user guide", "delfin", "api-clients"] 
+---
+
+
+## Scope
+
+This document is to help third party clients who wants to integrate [Delfin](https://github.com/sodafoundation/architecture-analysis/blob/master/arch-design/delfin/SODA_InfrastructureManagerDesign.md) into their Storage Management system
+
+## Goals
+
+ - To capture changes required by third party softwares to interface with Delfin APIs and use Delfin services to manage heterogeneous storage backends.
+
+## Non Goals
+
+- To document steps for Delfin developer for feature enhancements
+- To document third party driver development (Refer Third party driver development guide) for support more backends
+
+## Prerequisite
+
+- Ubuntu 16.04+
+- Python 3.6+
+- Delfin and its dependencies [installed](https://docs.sodafoundation.io/soda-gettingstarted/installation)
+(Delfin supports Ansible and Docker based installation for its deployment with all dependencies)
+
+## Configurations
+
+Delfin and its features/services are configured using configuration file `<delfin source path>/etc/delfin/delfin.conf`.
+
+
+Important configurations that users want change may be,
+
+- Database path
+- Enable/Disable metrics exporters
+- IP and Ports exporters
+- Cryptor used for encrypt sensitive data
+- Certificate file paths to interface with storage backends
```

Comment:
Here we can give a note: Delfin installation do not cover installation of client platform of are exporting to ( like Prometheus and alert manager) . User has to either install it manullay or choose default srm_tools from overall soda installer.

Reply:
Added a 'installation note' in prerequisite